openpype/hosts/max/plugins/create/create_review.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""Creator plugin for creating review in Max."""
+from openpype.hosts.max.api import plugin
+from openpype.pipeline import CreatedInstance
+from openpype.lib import BoolDef, EnumDef, NumberDef
+
+
+class CreateReview(plugin.MaxCreator):
+    """Review in 3dsMax"""
+
+    identifier = "io.openpype.creators.max.review"
+    label = "Review"
+    family = "review"
+    icon = "video-camera"
+
+    def create(self, subset_name, instance_data, pre_create_data):
+
+        instance_data["imageFormat"] = pre_create_data.get("imageFormat")
+        instance_data["keepImages"] = pre_create_data.get("keepImages")
+        instance_data["percentSize"] = pre_create_data.get("percentSize")
+        instance_data["rndLevel"] = pre_create_data.get("rndLevel")
+
+        _ = super(CreateReview, self).create(
+            subset_name,
+            instance_data,
+            pre_create_data)  # type: CreatedInstance
```

Comment:
If you're not using the returned data no need to capture it.
```suggestion
        super(CreateReview, self).create(
            subset_name,
            instance_data,
            pre_create_data)  # type: CreatedInstance
```

Reply:
for some reason `# type=CreatedInstance` will show red-lined underneath after removing  `_=`